bookwyrm/models/status.py

```diff
@@ -136,10 +136,18 @@ def ignore_activity(cls, activity):  # pylint: disable=too-many-return-statement
         # keep notes if they mention local users
         if activity.tag == MISSING or activity.tag is None:
             return True
-        tags = [l["href"] for l in activity.tag if l["type"] == "Mention"]
+
+        # BUG: this fixes the TypeError but if there is only one user mentioned
+        # we still don't get any notifs and DMs are dropped.
+        tags = activity.tag if type(activity.tag) == list else [ activity.tag ]
```

Comment:
This change stops Bookwyrm from throwing a TypeError but doesn't actually resolve the problem of not knowing who the post mentions if it only mentions a single person. Any tips appreciated.